packages/storybook/integration/keyboard.test.ts

```diff
@@ -5,64 +5,26 @@ describe('Keyboard Navigation', () => {
   let browser;
   const waitFnOptions = { timeout: 500 };
 
-  beforeAll(async (done) => {
+  beforeAll(async () => {
```

Comment:
@dtassone done is useless when using async

Reply:
moved to the "batch" pull request as this change isn't directly linked to the problem